app/mobile-app/src/yarn-error.log

```diff
@@ -0,0 +1,17648 @@
+Arguments: 
```

Comment:
We might want to squash merge to purge this from the repo history so it doesn't get bloated